### DISCUSSION TITLE
No longer able to see waypoint icons/colors (<SYM> tag)

### DISCUSSION THREAD

**Discussion by gbrown2036** [osmandapp/OsmAnd#19820] [Q&A]
A few years ago I was able to view the custom icon/color of specific waypoints in OsmAnd but now they all look the same.  The custom icons/colors are in my `<SYM>` tag (e.g. `Pin, Green` vs `Triangle, Blue`).

Is this a known bug or do I need to use different values for the <SYM> tag now?  If the latter, where do I find the list of valid values?

Below is the XML structure of two of my waypoints in a GPX file. 

`<wpt lat="84.000000" lon="-84.000000"><ele>-1712.671508799575718</ele><time>2023-06-15T10:26:26Z</time><name>~GREATSTRUCTURE</name><cmt/>**<sym>Pin, Green</sym>**<type>user</type></wpt>`

`<wpt lat="27.000000" lon="-83.000000"><ele>-51.304954499457456</ele><time>2023-06-15T10:26:26Z</time><name>X029</name><cmt>BS JUNE 2018 GOOD SH  BS JUNE 2018 GOOD SH-BS JUNE 2018 GOOD SH</cmt>**<sym>Triangle, Blue</sym>**<type>user</type></wpt>`

Thanks!

**Comment by dmpr0** [comment]
OsmAnd doesn't support <SYM> tag. OsmAnd store icons and color in custom <extension> tag https://docs.osmand.net/docs/user/personal/favorites#favorites-in-gpx-file
